deploy/operator-local.yaml

```diff
@@ -56,7 +56,7 @@ spec:
             - name: RELATED_IMAGE_pvc_jobs
               value: registry.access.redhat.com/ubi8-minimal:8.3-201
             - name: RELATED_IMAGE_postgres
-              value: centos/postgresql-96-centos7:9.6
+              value: registry.centos.org/centos/postgresql-96-centos7
```

Comment:
call me crazy but don't we want to use a tag here instead of :latest ?I know the project is fast-and-loose and bleeding-edge, but do you want unexpected changes here?

Reply:
Agree